core/src/main/java/org/apache/calcite/runtime/SqlFunctions.java

```diff
@@ -401,6 +402,16 @@ private static int makeRegexpFlags(@Nullable String stringFlags) {
     return flags;
   }
 
+  /** SQL {@code REGEXP_CONTAINS(value, regexp)} function. */
```

Comment:
Can you briefly document what happens if the regex is invalid.

Reply:
Added a statement, please check if this is what you had in mind.